website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -18,6 +18,34 @@ User accounts belong to individual people. Each user can be part of one or more
 Use the [Account API](/terraform/cloud-docs/api-docs/account) to get account details, update account information, and change your password.
 
 <!-- BEGIN: TFC:only -->
+## User Session Overview
+
+There are two types of Terraform accounts, standalone Terraform Cloud accounts and Terraform Cloud accounts linked to HCP accounts. 
+
+### Idle session timeout
+
+Terraform Cloud user sessions are automatically terminated if there has been no end user activity for a certain period of time. 
+
+- Standalone Terraform Cloud accounts can remain idle and valid for up to 14 days by default
+- Terraform Cloud accounts linked to an HCP account follow the HCP defaults and can remain idle for 1 hour by default
+
+After your session is terminated, you can resume your session by logging back in through accessing the Terraform Cloud TFC portal.  This is a security measure to prevent unauthorized access of unmonitored devices.
```

Comment:
```suggestion
After Terraform Cloud terminates a session, you can resume it by logging back in through the Terraform Cloud portal.  This is a security measure to prevent unauthorized access to unmonitored devices.
```